definitions/features/sync_plans.rb

```diff
@@ -47,6 +47,7 @@ def update_records(ids, enabled)
   ensure
     new_data = sync_plan_data(enabled, updated_record_ids)
     save_state(new_data)
+    updated_record_ids
```

Comment:
@iNecas , 
Again my bad, sorry!!
Updated the PR, I forgot method returns ids from each loop not updated_record_ids and thats why it is executed successfully.